db/src/main/kotlin/edu/artic/db/AppConverters.kt

```diff
@@ -149,12 +147,12 @@ class AppConverters {
     }
 
     @TypeConverter
-    fun fromDateTime(localDateTime: LocalDateTime?): Long? = localDateTime?.let {
-        it.toInstant(ZoneId.systemDefault().rules.getOffset(it)).toEpochMilli()
+    fun fromDateTime(input: ZonedDateTime?): String? = input?.let {
+        it.toString()
     }
 
     @TypeConverter
-    fun toDateTime(time: Long?): LocalDateTime? = time?.let { LocalDateTime.ofInstant(Instant.ofEpochMilli(it), ZoneId.systemDefault()) }
+    fun toDateTime(time: String?): ZonedDateTime? = time?.let { DateTimeFormatter.ISO_DATE_TIME.parse(it, ZonedDateTime.FROM) }
```

Comment:
might be worth doing a `try..catch datetimeparseexception` I think it throws? but we should also just log it if it does, but to be safe we dont crash here.

Reply:
I will check the date validity in Moshi adapter. If we catch the invalid dates when we are serializing we don't need it here. Right ?